python/servo/try_parser.py

```diff
@@ -18,24 +18,7 @@
 import logging
 
 from dataclasses import dataclass
-from enum import Enum, Flag, auto
```

Comment:
As mentioned [here](https://github.com/servo/servo/pull/36047#issuecomment-2746816228) the unused import is causing lint check to fail. We can see this by running `./mach test-tidy` locally. The CI won't allow the PR to be merge without fixing this.